src/app/cube/browse/browse.component.scss

```diff
@@ -309,3 +309,7 @@ $learning-object-width: 820px;
     }
   }
 }
+
```

Comment:
Looks pretty good! Just a couple of quick things.
Make sure you make the background white behind the X, and I'd recommend decreasing the margin between the X and the Sort item by about 20px.